docs/API.md

```diff
@@ -239,3 +239,17 @@ any.date();
 
 // 10/24/2028
 ```
+
+## `subset`
+
+Direct usage of [`pickset()` from chance.js](http://chancejs.com/#pickset).
+Options are passed directly to the chance method so refer to its documentation
+for what is available.
+
+### `subset` example
+
+```js
+any.subset(['one', 'two', 'three', 'four'], 2);
```

Comment:
i wonder if `any.subset(['one', 'two', 'three', 'four'], {size: 2});` might be more consistent with the patterns used elsewhere in `any`. don't think i have strong feelings about it though. do you have thoughts about that?